.mailmap

```diff
@@ -1238,6 +1238,7 @@ alijosephine <alijosephine@gmail.com>
 amsuhane <ayushsuhane99@iitkgp.ac.in> amsuhane <“ayushsuhane99@iitkgp.ac.in”>
 anca-mc <anca-mc@users.noreply.github.com>
 andreo <andrey.torba@gmail.com>
+anutosh491 <87052487+anutosh491@users.noreply.github.com>
 anutosh491 <andersonbhat491@gmail.com> Anutosh Bhat <87052487+anutosh491@users.noreply.github.com>
```

Comment:
This is listing you as two separate authors.

Reply:
Sure, will fix this